### PR TITLE
feat(js): Add support for formatting LangChain prompts as OpenAI and Anthropic payloads

### PR DESCRIPTION
**Pull request by jacoblee93**

### CODE REVIEW

File: js/src/utils/prompts/tests/openai.int.test.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  const { messages } = convertPromptToOpenAI(formattedPrompt);
+
+  const openAIClient = new OpenAI();
+
+  const openAIResponse = await openAIClient.chat.completions.create({
+    model: "gpt-4o-mini",
+    messages,
+  });
+
+  console.log(openAIResponse);
```
Comment (hinthornw) [inline js/src/utils/prompts/tests/openai.int.test.ts:21]: could we do something other than `logging`? it makes it kinda hard to sift through the tests

File: js/src/utils/prompts/tests/anthropic.int.test.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import Anthropic from "@anthropic-ai/sdk";
+import { pull } from "langchain/hub";
+
+import { convertPromptToAnthropic } from "../anthropic.js";
+
+test("basic traceable implementation", async () => {
+  const prompt = await pull("jacob/joke-generator");
```
[the remark this answers is not in the document]
Comment (jacoblee93) [reply js/src/utils/prompts/tests/anthropic.int.test.ts:7]: WDYM?

[the remark this answers is not in the document]
Comment (jacoblee93) [reply js/src/utils/prompts/tests/anthropic.int.test.ts:7]: Optics and consistency with Python?

**Comment by jacoblee93** [comment]
Moving to integration packages as discussed

**Pull request closed without merging**